plugins/module_utils/network/sonic/argspec/bgp_neighbors/bgp_neighbors.py

```diff
@@ -148,6 +148,24 @@ def __init__(self, **kwargs):
                                             },
                                             'type': 'dict'
                                         },
+                                        'ip_afi': {
+                                            'options': {
+                                                'default_policy_name': {'type': 'str'},
+                                                'send_default_route': {'default': False, 'type': 'bool'}
+                                            },
+                                            'type': 'dict'
+                                        },
+                                        'prefix_limit': {
+                                            'options': {
+                                                'max_prefixes': {'type': 'int'},
+                                                'prevent_teardown': {'default': False, 'type': 'bool'},
+                                                'warning_threshold': {'type': 'int'},
+                                                'restart_timer': {'type': 'int'}
+                                            },
+                                            'type': 'dict'
+                                        },
+                                        'prefix_list_in': {'type': 'str'},
```

Comment:
bgp neighbors afi safi is in bgp_neighbors_af module so I will be adding all of the afi safi prefix related attributes there

Reply:
Okay. (We need to fix this resource module structuring in a future release. It will confuse customers.)

For now, I think we can go ahead with getting the 2.0.0 functionality in on with the current structuring.